cmd/verify/permissions/cmd.go

```diff
@@ -64,6 +64,8 @@ func run(cmd *cobra.Command, argv []string) {
 	if err != nil {
 		reporter.Errorf("Unable to validate SCP policies")
 		reporter.Errorf("%v", err)
+		reporter.Infof("Verify your user's AWS account permissions")
```

Comment:
It would be better if this error informed the user that their specific IAM user doesn't have the correct permissions and to add the "AdministratorAccess" policy to it. We could even refer to the ARN `arn:aws:iam::aws:policy/AdministratorAccess`. I don't think we should provide the CLI steps as a user my not be able to add that policy to their user if they don't have the access already.

Reply:
The last sentence doesn't make much sense but what I'm trying to say is if they have been given an account and it doesn't have the AdministratorAccess policy attached to it, its unlikely they will be able to add it if they are not the account owner.

Reply:
Right that's why I only put "verify permissions" 
we don't know if the current user can actually those permissions for themselves, that's why i kept it generic